Option for Mixing_extruder

radek8 opened the pull request:
If the Mixing_extruder option is selected in Marlin, the printer uses two extruders, but reports only one extruder at the M115 command.

The FW display overwrites the manually entered number of extruders with what the printer reports.
This option disables the value override, and the value specified in Configuration.h is used

resolves #1055
resolves #1043

ETE-Design commented:
So with this you can also use 3 extruders for mixing?

radek8 commented:
> So with this you can also use 3 extruders for mixing?
> And is there also the options you have on an LCD with mixing hotend?

In the configuration.h file you define the number of Extruders (#define EXTRUDER_NUM)
However, this value will not be used if the printer reports the number of extruders on the M115 command.
If the #define MIXING_EXTRUDER value is enabled in Marlin
Marlin reports only 1 (virtual) extruder
Enabling the MIXING_EXTRUDER value in the TFT will not use the value obtained by the M115 command, but will use the user-defined value.

SerjPr commented:
> > Значит, при этом вы также можете использовать 3 экструдера для смешивания? 
> > И есть ли у вас еще опции на ЖК-дисплее с хотэндом для микширования?
> 
> В файле configuration.h вы определяете количество экструдеров (#define EXTRUDER_NUM).
> Однако это значение не будет использоваться, если принтер сообщает количество экструдеров по команде M115.
> Если значение #define MIXING_EXTRUDER включено в Marlin
> Marlin сообщает только 1 (виртуальный) экструдер.
> Включение значения MIXING_EXTRUDER в TFT не будет использовать значение, полученное командой M115, но будет использовать значение, определенное пользователем.

I will make changes to the code in the "parseACK.c " "Configuration.h" files.
The option to select a second extruder in the extrusion menu has worked. Unloading and loading filament is working. But an unnecessary inscription appeared on the screen. Previously, the Status screen displayed the printer name"Longer Serj". Which I entered in the marlin settings. And now "Longer Serj EXTRUDER_COUNT: 1 UUID: cede2a2f-41a2-4748-9b12-c55c62f367ff"

radek8 commented:
> > > ,Начит, при этом вы также можете использовать 3 экструдера для смешивания? 
> > > И есть ли у вас еще опции на ЖК-дисплее с хотэндом для микширования?
> > 
> > 
> > В файле configuration.h вы определяете количество экструдеров (#define EXTRUDER_NUM). 
> > Однако это значение не будет использоваться, если принтер сообщает количество экструдеров по кома 
> > #Сли значение #define MIXING_EXTRUDER включено в Marlin 
> > Marlin сообщает только 1 (виртуальный) экструдер. 
> > Включение значения MIXING_EXTRUDER в TFT не будет использовать значение, полученное командой оос о
> 
> Provedu změny v kódu v souborech „parseACK.c“ „Configuration.h“.

8/5000
fixed Možnost vybrat druhý extruder v nabídce vytlačování fungovala. Vykládání a vkládání vlákna funguje. Na obrazovce se ale objevil zbytečný nápis. Dříve se na obrazovce Stav zobrazoval název tiskárny „Longer Serj“. Který jsem zadal v nastavení marlin. A nyní „Delší Serj EXTRUDER_COUNT: 1 UUID: cede2a2f-41a2-4748-9b12-c55c62f367ff“


8/5000
fixed

SerjPr commented:
> bad name fixed

Made changes to the code. The information on the display is now correct. Thanks a lot!
